button.py

```diff
@@ -8,8 +8,8 @@ def __init__(self, x, y, image_path, action, text=None):
         self.graphic = pygame.image.load(image_path)
         self.rect = self.graphic.get_rect()
 
-        self.x = x
-        self.y = y
+        self.x = x - self.rect.width / 2
+        self.y = y - self.rect.height / 2
```

Comment:
Could you compare with floor division and see how that affects the dimensions?

Also look at comparing other int divisions too.